lottie-ios/Classes/AnimatableLayers/LOTEllipseShapeLayer.m

```diff
@@ -10,7 +10,7 @@
 #import "CAAnimationGroup+LOTAnimatableGroup.h"
 #import "LOTStrokeShapeLayer.h"
 
-const CGFloat kEllipseControlPointPercentage = 0.55228;
+static const CGFloat kEllipseControlPointPercentage = 0.55228;
```

Comment:
This was caused by `-Wmissing-variable-declarations`, saying `No previous extern declaration for non-static variable`.